Input Notebook:  a_send_gpt_buches.ipynb
Output Notebook: a_send_gpt_buches_F_20251001_143252.ipynb
Executing:   0%|          | 0/12 [00:00<?, ?cell/s][IPKernelApp] WARNING | Unknown error in handling PYTHONSTARTUP file /home/marcelino.maita/.vscode-server/data/User/workspaceStorage/88754bb242e7b47f22d0b0905e35f833/ms-python.python/pythonrc.py:
Executing notebook with kernel: python3
Executing:   8%|▊         | 1/12 [00:02<00:30,  2.80s/cell]Executing:  25%|██▌       | 3/12 [00:03<00:09,  1.06s/cell]Executing:  50%|█████     | 6/12 [00:06<00:06,  1.08s/cell]Executing:  58%|█████▊    | 7/12 [00:10<00:08,  1.64s/cell]Executing:  83%|████████▎ | 10/12 [00:13<00:02,  1.24s/cell]Executing:  92%|█████████▏| 11/12 [57:03<10:48, 648.40s/cell]Executing: 100%|██████████| 12/12 [57:03<00:00, 285.33s/cell]
